src/Driver/AbstractMySQLDriver.php

```diff
@@ -46,7 +47,14 @@ public function getDatabasePlatform(ServerVersionProvider $versionProvider): Abs
             return new MariaDBPlatform();
         }
 
-        if (version_compare($version, '8.0.0', '>=')) {
+        if (version_compare($version, '8.0.13', '>=')) {
+            return new MySQL8013Platform();
+        }
+
+        if (
+            version_compare($version, '8.0.0', '>=')
+            && version_compare($version, '8.0.13', '<')
+        ) {
             return new MySQL80Platform();
```

Comment:
Please deprecate `MySQL80Platform` and trigger a proper deprecation notice if it is picked here. We should not support 8.0.13 anymore in DBAL 5.

Reply:
done

Reply:
Depends on when we release it. We usually drop support for all EOL versions when we do a new major.